missioncontrol/settings.py

```diff
@@ -98,7 +98,7 @@
 
 PRESTO_URL = config('PRESTO_URL')
 MISSION_CONTROL_TABLE = config('MISSION_CONTROL_TABLE',
-                               default='telemetry.error_aggregates')
+                               default='telemetry.error_aggregates_v1')
```

Comment:
Assuming we use this default in production, this change should fix things there too (when error_aggregates_v1 points to the right table, in any case).

Reply:
We don't use the default. It should already be set explicitly: https://github.com/mozilla-services/cloudops-deployment/compare/missioncontrol_api?expand=1#diff-ee3c8f1eb0c25528567bd976d40550c3R16

Reply:
I can't see that, what does it say for the value? Do we need to change it there?

Regardless we should probably apply this for local development purposes.